src/main/java/school/lemon/changerequest/java/vehicles/Boat.java

```diff
@@ -35,39 +35,33 @@ public void setMotorized(boolean motorized) {
      * for airplanes: This airplane is a 1998 ABC Motors Comac that can reach 10000 feet.
      */
     public String toString() {
-        String result = String.format("This boat is a %d %s %s",
-                manufacturedYear, manufacturer, model);
+        StringBuffer result = new StringBuffer(String.format("This boat is a %d %s %s",
```

Comment:
Take a look at the difference between `StringBuilder` and `StringBuffer`. You have to use `StringBuilder` in 99.9% of cases.

Also, while you are already using dynamic strings — there is no need for `String.format` anymore, as inside `String.format` there is custom implementation of dynamic strings + parsing of the format and other stuff — just use chained calls of `.append` on your dynamic string and add all the needed parts of the text.